staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -270,7 +303,7 @@ func (s *store) Delete(
 		return fmt.Errorf("unable to convert output object to pointer: %v", err)
 	}
 
-	skipTransformDecode := false
```

Comment:
this is blowing my mind right now.  We already had this and hardcoded it false?

Reply:
no, we added this as a separate refactor PR: https://github.com/kubernetes/kubernetes/pull/127778